### PR TITLE
NOISSUE: No logs for the dead node.

### PR DESCRIPTION
**Pull request by Vany**
Stop logging after fatal

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by ita-sammann** [comment]
Dead node write no logs

File: log/log_test.go [hunk 1]
```diff
@@ -64,6 +66,8 @@ func TestLog_GlobalLogger(t *testing.T) {
 	assert.Panics(t, func() { capture(func() { Panic("HelloWorld") }) })
 	assert.Panics(t, func() { capture(func() { Panicf("%s", "HelloWorld") }) })
 
+	zerolog.SetGlobalLevel(zerolog.DebugLevel)
```
Comment (dmitry-panchenko) [inline log/log_test.go]: Could you comment this line please?

Comment (Vany) [reply log/log_test.go]: nope. it will broke test

Comment (dmitry-panchenko) [reply log/log_test.go]: Sorry. I mean add comment.

Comment (Vany) [reply log/log_test.go]: I understand what do you want. =)

**Pull request merged**